lib/internal/child_process.js

```diff
@@ -224,19 +230,19 @@ function stdioStringToArray(stdio, channel) {
 
   switch (stdio) {
     case 'ignore':
-    case 'pipe': options.push(stdio, stdio, stdio); break;
-    case 'inherit': options.push(0, 1, 2); break;
+    case 'pipe': ArrayPrototypePush(options, stdio, stdio, stdio); break;
+    case 'inherit': ArrayPrototypePush(options, 0, 1, 2); break;
     default:
       throw new ERR_INVALID_ARG_VALUE('stdio', stdio);
   }
 
-  if (channel) options.push(channel);
+  if (channel) ArrayPrototypePush(options, channel);
 
   return options;
 }
 
 function ChildProcess() {
-  EventEmitter.call(this);
+  ReflectApply(EventEmitter, this, []);
```

Comment:
Since https://github.com/nodejs/node/pull/36221, `FunctionPrototypeCall` is now as efficient as `ReflectApply`. Here, because there's no arguments to pass to `EventEmitter`, you can avoid creating an empty array thus gaining some marginal perf improvement.
```suggestion
  FunctionPrototypeCall(EventEmitter, this);
```
If you did that because of the email exchange we had, sorry for misguiding you, until recently that was indeed the most efficient way of doing this.